src/format/getFormattedFailures.ts

```diff
@@ -0,0 +1,13 @@
+import stripAnsi from 'strip-ansi';
+
+import { JsonReport } from '../typings/JsonReport';
+
+export const getFailureDetails = (report: JsonReport): string =>
+    report.testResults &&
+    report.testResults.some(({ message }) => message.length > 0)
+        ? '\n```bash\n' +
+          report.testResults
+              ?.map(({ message }) => stripAnsi(message))
+              .join('```\n---\n```bash\n') +
+          '```'
+        : '';
```

Comment:
Could you take a look at #132? This PR describes and fixes some issues with such error format.

Reply:
Done in https://github.com/ArtiomTr/jest-coverage-report-action/pull/137/commits/0478397a7b51219c415389670ec49c97714a06c1